final_task/rss_reader/pdf_converter.py

```diff
@@ -54,6 +56,11 @@ def convert_to_pdf(articles_list: list, path: str, rss_url: str) -> None:
                     try:
                         filename, headers = urllib.request.urlretrieve(link[0])
                         image_type = headers['content-type'].replace('image/', '')
+                        if image_type not in ['jpg', 'jpeg', 'png', 'JPG', 'JPEG', 'PNG']:
+                            pdf.multi_cell(w=0, h=5, txt=f'[{number}]: {link[0]}')
+                            pdf.ln(5)
+                            continue
```

Comment:
It is better to avoid using `continue` as it lowers readability. In this specific situation this is ok, but if you will tryto add more complex logic here it will quickly become unreadable. For example here you can use `else` for the block after `continue`.